pages/portfolio/index.vue

```diff
@@ -58,68 +58,76 @@
       </client-only>
     </div>
     <main
-      class="w-100% bg-hex-101010 z-1 p-10"
+      class="w-full% bg-hex-101010 z-1 p-10"
     >
-      <div class="nav flex p-10">
-        <div class="nav-elements w-1/2 p-8 drop-shadow-3xl">
-          <lottie
-            :width="250"
-            :height="250"
-            :options="lottieOptions.first"
-            class="lottie"
-            @animCreated="handleAnimation($event, 'firstAnim')"
-          />
-          <span class="text-3xl mt-10 py-10">Developers</span>
-          <p class="text-sm mt-10">
-            A team of zealous coders improvising the world as you complete reading this sentence.
-          </p>
-          <button
-            class="
-              btn
-              w-30
-              p-2
-              mt-10
-              border
-              rounded-lg
-              hover:bg-hex-fff hover:text-black
-            "
-            aria-label="view-team"
-            @click="showSdg()"
-          >
-            View Team
-          </button>
-        </div>
-        <div class="nav-elements w-50vw p-8 -mt-20">
-          <lottie
-            :width="250"
-            :height="250"
-            :options="lottieOptions.sophies"
-            class="lottie"
-            @mouseover="play"
-            @focus="play"
-            @animCreated="handleAnimation($event, 'sophiesAnim')"
-          />
-          <span class="text-3xl mt-10 py-10">Alums</span>
-          <p class="text-sm mt-10">
-            They are the building blocks, our mentors, and the very soul of this team.
-          </p>
-          <button
-            class="
-              btn
-              w-30
-              p-2
-              mt-10
-              border
-              font-400
-              rounded-lg
-              hover:bg-hex-fff hover:text-black
-            "
-            aria-label="view-alums"
-            @click="showAlums()"
-          >
-            View Team
-          </button>
-        </div>
+      <div class="nav flex-col lg:flex-row flex p-10 justify-evenly">
```

Comment:
There is some responsiveness issue on the Teams page for the lottie containers, I guess removing the `flex-col` for small-screen would fix this.

Reply:
the css was overlapping with text classes removed it for now